documentation/src/main/asciidoc/topics/proc_enabling_jmx_port.adoc

```diff
@@ -11,23 +11,42 @@ If you want to remotely access {brandname} Server via JMX you must enable a remo
 ====
 endif::remote_caches[]
 
-.Procedure
+You can enable remote JMX ports using one of the following approaches:
```

Comment:
I would differentiate this even further:
* Enable remote JMX requiring authentication to one of the Infinispan server security realms
* Enable remote JMX manually, using the standard Java management configuration options

Then, when describing the latter, we can also say that it can be used to set up unsecured JMX remoting, which is not recommended.